valuesets/ValueSet-UKCore-AllergyCode.xml

```diff
@@ -32,26 +32,22 @@
 &#13; - AMP – Actual Medicinal Product
 &#13; - Ingredient &#13;&#13;
 
-A set of codes from the SNOMED CT UK coding system that:
-
-&#13; - identify a substance or pharmaceutical or biologic product that the patient has an allergy or intolerance to
-&#13; - state that the patient has no known allergy or does not have a specific allergy
-&#13; - provide a degrade of information about a drug or non-drug allergy &#13;&#13;
+A set of codes from the SNOMED CT UK coding system that:  
+&#13; - identify a substance or pharmaceutical or biologic product that the patient has an allergy or intolerance to 
+&#13; - state that the patient has no known allergy or does not have a specific allergy 
+&#13; - provide a degrade of information about a drug or non-drug allergy 
 
 Where no dm+d or SNOMED coded information is available, a specific code from the nullFlavor Code System can be used instead to indicate this.
 
-"/>
+	"/>
 	<copyright value="Copyright &#169; 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.&lt;br$gt;This value set includes content from SNOMED CT, which is copyright &#169; 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement." />
 	<compose>
-		<include>
-			<system value="https://dmd.nhs.uk"/>
-		</include>
 		<include>
 			<system value="http://snomed.info/sct"/>
 			<filter>
 				<property value="constraint"/>
 				<op value="="/>
-				<value value="descendantOf 105590001 OR descendantOf 373873005 OR descendantOrSelfOf 716186003 OR 196461000000101 OR 196471000000108"/>
+				<value value="memberOf 999000581000001102 OR memberOf 999000561000001109 OR memberOf 999000541000001108 OR memberOf 999000791000001109 OR descendantOf 105590001 OR descendantOf 373873005 OR descendantOrSelfOf 716186003 OR 196461000000101 OR 196471000000108"/>

```

Comment:
Can we add the following two concepts as descendentOrSelfOf
418634005 | Allergic reaction caused by substance (disorder) 
293104008 | Adverse reaction to component of vaccine product (disorder)

Reply:
Both added.